src/Generators/ModelGenerator.php

```diff
@@ -24,8 +26,12 @@ public function output(array $tree): array
 
         $stub = $this->files->stub('model/class.stub');
 
+        $this->registerModels($tree);
+
+        $this->mapPolymorphicRelationships();
+
         /** @var \Blueprint\Models\Model $model */
-        foreach ($tree['models'] as $model) {
+        foreach ($this->models as $model) {
```

Comment:
You don't want to loop over every model. This will regenerate "cached" models as well.